pyramid_mixpanel/query.py

```diff
@@ -4,6 +4,19 @@
 import typing as t
 
 
+class MultipleProfilesFoundException(Exception):
```

Comment:
Let's subclass from `ValueError` so that if any client code is catching `ValueError` the code keeps on working.